docs/modules/ROOT/pages/index.adoc

```diff
@@ -259,15 +324,28 @@ The schema for this type is defined by the `identity` rule in the following http
 ----
 identity = {
   "signer_payload": $signer-payload-map, ; content to be signed by credential holder
-  "sig_type": tstr .size (1..max-tstr-length), ; a string identifying the data type of the signature field that follows
   "signature": bstr, ; byte string of the signature
   "pad1": bstr, ; byte strings filled with binary `0x00` values used for filling up space
   ? "pad2": bstr, ; optional byte strings filled with binary `0x00` values used for filling up space
 }
 
 signer-payload-map = {
+  "sig_type": tstr .size (1..max-tstr-length), ; a string identifying the data type of the signature 
   "referenced_assertions": [1* $hashed-uri-map],
```

Comment:
I don't think we need `referenced` in here, we don't prefix that in a C2PA Claim, which is what this data structure essentially is.